packages/api/src/controllers/notificationUserController.js

```diff
@@ -13,16 +13,127 @@
  *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
  */
 
-const NotificationUserModel = require('../models/notificationUserModel');
+const NotificationUser = require('../models/notificationUserModel');
 
+/**
+ * Controls requests related to the user's notifications.
+ */
 module.exports = {
+  /**
+   * Establishes a subscription for the user's notifications.
+   * @param {Request} req - The HTTP request object.
+   * @param {Response} res - The HTTP response object.
+   */
+  subscribeToAlerts(req, res) {
```

Comment:
QQ, how long can a subscription last? I thought maybe websockets might be a better choice but if this does the trick I guess why bother.

Reply:
I'm not aware of any limits, either to the EventSource API or to the way they are used here. I do plan to add some code for monitoring/cleanup of long lasting subscriptions tracked by the server.

Reply:
We considered WebSockets. See https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1177026576/Near+Real+Time+Alerts